libsrc/ismrmrd.cpp

```diff
@@ -1216,24 +1216,24 @@ template <> EXPORTISMRMRD inline ISMRMRD_DataTypes get_data_type<complex_double_
 }
 
 // Images
-template EXPORTISMRMRD class Image<uint16_t>;
-template EXPORTISMRMRD class Image<int16_t>;
-template EXPORTISMRMRD class Image<uint32_t>;
-template EXPORTISMRMRD class Image<int32_t>;
-template EXPORTISMRMRD class Image<float>;
-template EXPORTISMRMRD class Image<double>;
-template EXPORTISMRMRD class Image<complex_float_t>;
-template EXPORTISMRMRD class Image<complex_double_t>;
+template class EXPORTISMRMRD Image<uint16_t>;
```

Comment:
Is this an intentional bugfix?

Reply:
Yes. Won't compile in VS 2022 otherwise.